### ISSUE TITLE
JAWS/NVDA is not announcing the tooltip info for the "Search Paths" "Script Arguments" "Interpreter Path" "Interpreter Arguments" etc., edit fields: A11y_Accessibility Testing for Python in VS_Project_4294967294_Properties_ScreenReader

### ISSUE THREAD

**Issue by cwebster-99** [microsoft/PTVS#7849]
https://devdiv.visualstudio.com/DevDiv/_queries/query/?tempQueryId=5dc9c31e-c7b7-4f49-9513-c26952986538

> Note: Must be addressed in 60 days in order to stay compliant. 

**Actual:**
JAWS/NVDA is not announcing the tooltip info for the "Search Paths" "Script Arguments" "Interpreter Path" "Interpreter Arguments" etc...  edit fields.

**Expected:**
JAWS/NVDA should announce the tooltip info for the "Search Paths" "Script Arguments" "Interpreter Path" "Interpreter Arguments" etc...  edit fields.

**Environment Details:**
Application Name: Accessibility Testing for Python in VS
Edition Windows 11 Enterprise
Version 22H2
OS build 22621.3155
Windows Feature Experience Pack 1000.22684.1000.0
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.9.2
Screen Reader: JAWS (2024.2403.3), NVDA (2023.2.0.29051)

**Repro Steps:**
Open visual studio.
Start JAWS.
TAB till Create a new Project and press ENTER key.
Create a "project" dialog will open.
Search "Flask Web" project and select it.
TAB till next button and press ENTER key.
Configure your new project screen will open.
Provide project name.
TAB to create button and press enter.
Project will be created.
TAB till project menu item and press ENTER key.
TAB till 4294967294_Properties and press ENTER key.
Now observe the issue.
